content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Saving
+---
+
+import {Box, Text} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    Save patterns help users store and update their content and configuration throughout GitHub. These changes should be represented in the UI accurately, quickly, and obviously. Their behavior should inspire confidence and trust.
+</Box>
+
+## Calls to action
+
+When data is not automatically saved after the user make changes, buttons are used to submit or cancel the changes.
+
+### Text
+
+All configuration flows should have calls to action that include an obvious active verb such as "Create", "Save", "Delete", or "Update" (for example, "Create" a new repository, "Add" an SSH key to your profile, "Save" security preferences, "Update" a repository's description, "Delete" an old email address).
+
+When defining these calls to actions, make sure to:
+1. Keep the text as succinct and clear as possible
+2. Add the item’s name to the button text in cases where it may be unclear to the user what is being changed
+
+<DoDontContainer>
+  <Do>
+    <img src="https://user-images.githubusercontent.com/2313998/152432682-2e102919-114a-43ce-a446-5501d287084a.png" />
+    <Caption>Use descriptive text with an active verb</Caption>
```

Comment:
Are these images swapped?

Reply:
+1, I was confused because the image on the right seems like the right call, but it's a 'dont' 🤔

Reply:
Yup, these are swapped. Silly mistake on my part 🙈